KatanaTests/Storage/Core/ActionLinkerTests.swift

```diff
@@ -0,0 +1,477 @@
+//
+//  ActionLinkerTests.swift
+//  Katana
+//
+//  Created by Riccardo Cipolleschi on 12/01/2017.
+//  Copyright © 2017 Bending Spoons. All rights reserved.
+//
+import Foundation
+import XCTest
+@testable import Katana
+
+class ActionLinkerTests: XCTestCase {
+  
+  override func setUp() {
+    super.setUp()
+  }
+    
+  override func tearDown() {
+    super.tearDown()
+  }
+
+  //MARK Creation tests
+  func testCreationEmpty() {
+    let actionLinker = ActionLinker(links: [])
+    XCTAssertTrue(actionLinker.links.isEmpty)
+  }
+  
+  func testCreationOneSourceActionOneLink() {
+    let actionLinker = ActionLinker(links: [ActionLinks(source: BaseAction.self, links: [LinkedAction1.self])])
+    XCTAssertEqual(actionLinker.links.count, 1)
+    XCTAssertEqual(actionLinker.links[String(reflecting:BaseAction.self)]!.count, 1)
```

Comment:
Instead of `String(reflecting)` you should use the method provided by ActionLinker (which could be also static, btw).

You don't want to test how things are stored, but just that they are stored